File: Facebook.NET/GraphAPI.VideoUploadLimit.cs
using System;
using System.Runtime.Serialization;

namespace Facebook
{
    public static partial class GraphAPI
    {
        [DataContract]
        public sealed class VideoUploadLimit
        {
            [DataMember(Name = "length")]
            public double? Length { get; set; }

            [DataMember(Name = "size")]
            public long? Size { get; set; }
        }
    }
}

File: Facebook.NET - VB/Contracts/GraphAPI.VideoUploadLimit.vb
Imports System
Imports System.Runtime.Serialization

Namespace Facebook
    Public NotInheritable Partial Class GraphApi
        <DataContract()>
        Public NotInheritable Class VideoUploadLimit
            <DataMember(Name:="length")>
            Public Property Length() As System.Nullable(Of Double)
            <DataMember(Name:="size")>
            Public Property Size() As System.Nullable(Of Long)
        End Class
    End Class
End Namespace